ci-operator/step-registry/redhat-chaos/pod-network-chaos/observer/redhat-chaos-pod-network-chaos-observer-commands.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python3 --version
+ls
+
+while [ ! -f "${KUBECONFIG}" ]; do
+  printf "%s: waiting for %s\n" "$(date --utc --iso=s)" "${KUBECONFIG}"
+  sleep 30
+done
+printf "%s: acquired %s\n" "$(date --utc --iso=s)" "${KUBECONFIG}"
+
+echo "kubeconfig loc $KUBECONFIG"
+
+echo "kubeconfig loc $$KUBECONFIG"
+echo "Using the flattened version of kubeconfig"
+oc config view --flatten > /tmp/config
+export KUBECONFIG=/tmp/config
+
+while [ "$(oc get ns | grep -c 'start-kraken')" -lt 1 ]; do
+  echo "start kraken not found yet, waiting"
+  sleep 15
+done
+
+while [ "$(oc get network.config.openshift.io cluster -o jsonpath='{.status.networkType}')" != "OVNKubernetes" ]; do
```

Comment:
are we able to put this while loop in an **if** statement based on an environment variable? With the default to not run it but for you to set it to true for your specific use case